application/account-management/WebApp/package.json

```diff
@@ -1,27 +1,31 @@
 {
-  "name": "account-management-webapp",
-  "private": true,
-  "version": "1.0.0",
-  "scripts": {
-    "dev": "rspack serve",
-    "build": "rspack build"
-  },
-  "dependencies": {
-    "react": "^18.2.0",
-    "react-aria-components": "^1.0.0-beta.2",
-    "react-dom": "^18.2.0"
-  },
-  "devDependencies": {
-    "@rspack/cli": "latest",
-    "@types/react": "^18.2.33",
-    "@types/react-dom": "^18.2.14",
-    "autoprefixer": "^10.4.16",
-    "postcss": "^8.4.31",
-    "postcss-loader": "^7.3.3",
-    "prettier": "^3.0.3",
-    "tailwindcss": "^3.3.5",
-    "tailwindcss-animate": "^1.0.7",
-    "tailwindcss-react-aria-components": "^1.0.0-beta.1",
-    "typescript": "^5.2.2"
-  }
+    "name": "account-management-webapp",
+    "private": true,
+    "version": "1.0.0",
+    "scripts": {
+        "dev": "rspack serve",
+        "build": "rspack build"
+    },
+    "dependencies": {
+        "@tanstack/react-query": "^4.36.1",
+        "openapi-fetch": "^0.8.1",
+        "react": "^18.2.0",
+        "react-aria-components": "^1.0.0-beta.2",
+        "react-dom": "^18.2.0",
+        "zod": "^3.22.4"
+    },
+    "devDependencies": {
+        "@rspack/cli": "latest",
+        "@types/react": "^18.2.33",
+        "@types/react-dom": "^18.2.14",
+        "@platform-platform/account-management-api": "1.0.0",
+        "autoprefixer": "^10.4.16",
+        "postcss": "^8.4.31",
+        "postcss-loader": "^7.3.3",
+        "prettier": "^3.0.3",
+        "tailwindcss": "^3.3.5",
+        "tailwindcss-animate": "^1.0.7",
+        "tailwindcss-react-aria-components": "^1.0.0-beta.1",
+        "typescript": "^5.2.2"
+    }
```

Comment:
not sure why Rider changed the indention of a package.json file - even prettier knows better 🧐